bindings/src/main/java/io/pravega/storage/filesystem/FileSystemStorage.java

```diff
@@ -197,6 +197,14 @@ public void close() {
 
     //endregion
 
+    protected FileChannel getFileChannel(Path path, StandardOpenOption openOption) throws IOException {
```

Comment:
Please add `@VisibleForTesting` for both these methods.

Reply:
fixed.